src/cephadm/cephadmlib/file_utils.py

```diff
@@ -160,3 +161,26 @@ def unlink_file(
     except Exception:
         if not ignore_errors:
             raise
+
+
+def update_meta_file(file_path: str, update_key_val: dict) -> None:
+    """Update key in the file with provided value"""
+    try:
+        with open(file_path, 'r') as fh:
+            data = json.load(fh)
+        file_stat = os.stat(file_path)
+    except FileNotFoundError:
+        raise
+    except Exception:
+        logger.exception(f'Failed to update {file_path}')
+        raise
+    data.update(
+        {key: update_key_val[key] for key in update_key_val if key in data}
```

Comment:
NIT, I think this could be done with this could be done with `key: value for key, value in update_key_val.items() if key in data`

Reply:
addressed